### PR TITLE
feat: update to yearn version 1.0

### PR DESCRIPTION
**Pull request by steffenix**

### CODE REVIEW

File: contracts/Strategy.sol [hunk 1]
```diff
@@ -107,4 +105,26 @@ contract Strategy is BaseStrategy {
...
+     * @param _amtInWei The amount (in wei/1e-18 ETH) to convert to `want`
+     * @return The amount in `want` of `_amtInEth` converted to `want`
+     **/
+    function ethToWant(uint256 _amtInWei)
+        public
+        view
+        virtual
+        override
+        returns (uint256)
+    {}
```
Comment (fubuloubu) [inline contracts/Strategy.sol:129]: This might be a separate PR for 0.4.0 release

**Review by fubuloubu — APPROVED** [review]
Blocked on `v1.0.0` release (post-audit)

**Comment by steffenix** [comment]
Not gonna merge 0.8 changes for now.
